pandas/tests/io/parser/common.py

```diff
@@ -1603,3 +1604,16 @@ def test_buffer_rd_bytes_bad_unicode(self):
             t = TextIOWrapper(t, encoding='ascii', errors='surrogateescape')
         with pytest.raises(UnicodeError):
             pd.read_csv(t, encoding='UTF-8')
+
+    def test_EA_types(self):
```

Comment:
we want to try this generically, can you make a new module in pandas/tests/extension/base called io.py and follow the existing patterns.

Reply:
Yeah no problem.